conan/tools/microsoft/msbuilddeps.py

```diff
@@ -25,6 +26,7 @@ class MSBuildDeps(object):
             <Conan{name}BinaryDirectories>{bin_dirs}</Conan{name}BinaryDirectories>
             <Conan{name}Libraries>{libs}</Conan{name}Libraries>
             <Conan{name}SystemDeps>{system_libs}</Conan{name}SystemDeps>
+            <Conan{name}CAExcludeDirectories>{ca_exclude_dirs}</Conan{name}CAExcludeDirectories>
```

Comment:
This is not new "data" definition, shouldn't be defined here, but just in the group below, as an ``if ca_exclude_path``.

Reply:
not sure I follow - it uses the same structure as another definitions...

Reply:
got it, then I agree it shouldn't be there